pyodbc-4.0.28.dist-info/DESCRIPTION.rst,sha256=zvBd0s-GAWA7NMmwHWkdPbRGltkhjHUiLpGLzUYkVWk,154
pyodbc-4.0.28.dist-info/INSTALLER,sha256=zuuue4knoyJ-UwPPXg8fezS7VCrXJQrAP7zeNuwvFQg,4
pyodbc-4.0.28.dist-info/METADATA,sha256=jSb_2reHKMVLtFV55ASGaBLlgIhjq3Ooj8W9K8EOuwU,1056
pyodbc-4.0.28.dist-info/RECORD,,
pyodbc-4.0.28.dist-info/REQUESTED,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
pyodbc-4.0.28.dist-info/WHEEL,sha256=X8kVdBCq85ICewwfaE6btv5qKsFQfVq8NYJIXUK0i1A,104
pyodbc-4.0.28.dist-info/metadata.json,sha256=AUPHJIKLyzgPKFhYBojokcaFaOWmW6GIeZorrjfR45E,975
pyodbc-4.0.28.dist-info/top_level.txt,sha256=30jvYcCul6_wA26OT5c068GAHzqQLpd9WfQ0mSSsDTU,7
pyodbc.cpython-36m-x86_64-linux-gnu.so,sha256=2K-r0VxG9aly50tePbhgGmQkdMEQ9RtOCkRyLMiK8gg,684528
